test/wdio_functional/WDIO_FUNCTIONAL_TESTS.md

```diff
@@ -0,0 +1,269 @@
+# Kibana Test Harness
+
+## Running Tests
+
+### Production Command
+
+`yarn test:ui:wdio`
```

Comment:
As implemented, this doesn't start the servers right? Should it? Could we create `scripts/wdio_servers` that does something similar to what the `scripts/functional_tests_servers` script does?